src/main.ts

```diff
@@ -2,6 +2,7 @@ import * as path from 'path'
 import * as core from '@actions/core'
 import * as glob from '@actions/glob'
 import {promises as fs} from 'fs'
+import { escapeStringRegexp } from 'escape-string-regexp'
```

Comment:
🚫 **[eslint]** <[import/named](https://github.com/import-js/eslint-plugin-import/blob/v2.25.4/docs/rules/named.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>escapeStringRegexp not found in 'escape-string-regexp'